cli/src/pcluster/aws/aws_resources.py

```diff
@@ -253,7 +252,7 @@ def supported_architecture(self):
 
     def is_efa_supported(self):
         """Check whether EFA is supported."""
-        return self.instance_type_data.get("NetworkInfo").get("EfaSupported")
+        return self.instance_type_data.get("NetworkInfo", {}).get("EfaSupported")
```

Comment:
what about returning a default for "EfaSupported"? i.e. `.get("EfaSupported", false)`